lab_5_scrapper/scrapper.py

```diff
@@ -100,6 +217,12 @@ def make_request(url: str, config: Config) -> requests.models.Response:
     Returns:
         requests.models.Response: A response from a request
     """
+    headers = config.get_headers()
+    timeout = config.get_timeout()
+    verify = config.get_verify_certificate()
+    response = requests.get(url, headers=headers, timeout=timeout, verify=verify)
```

Comment:
you may write config.get_* directly inside requests.get call

no need to create separate variables for each config parameter above

Reply:
Done